recipe/meta.yaml

```
@@ -150,8 +150,10 @@ outputs:
       string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [cuda_compiler_version == "None"]
       detect_binary_files_with_prefix: False
       # weigh down cpu implementation and give cuda preference
-      track_features:
-        - pytorch-cpu                                      # [cuda_compiler_version == "None"]
+      track_features:                                           # [cuda_compiler_version == "None"]
+        - pytorch-cpu                                           # [cuda_compiler_version == "None"]
+      run_exports:                                              # [cuda_compiler_version != "None"]
+        - {{ pin_subpackage('pytorch', max_pin='x.x') }} cuda*  # [cuda_compiler_version != "None"]
```

Doesn't the CPU version need the same run-export?

```suggestion
      run_exports:                                              # [cuda_compiler_version != "None"]
        - {{ pin_subpackage('pytorch', max_pin='x.x') }} cpu*   # [cuda_compiler_version == "None"]
        - {{ pin_subpackage('pytorch', max_pin='x.x') }} cuda*  # [cuda_compiler_version != "None"]
```

---

I decided against this feature. upstream got rid of this package name.